source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,109 @@
+.. _cpp-single-field-index:
+
+====================
+Single Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Single field indexes** are indexes that hold a reference to a single field of a
+document in a collection. These indexes improve single field query and
```

Comment:
S: This wording kinda makes it seem like the index only applies to one document. You could reword to:
```suggestion
**Single field indexes** are indexes that hold a reference to a single field of a
collection's documents. These indexes improve single field query and
```